services/movie_session.py

```diff
@@ -0,0 +1,52 @@
+from datetime import datetime
+
+from django.db.models import QuerySet
+
+from db.models import MovieSession, CinemaHall
+from services.movie import get_movie_by_id
+
+
+def create_movie_session(
+    movie_show_time: datetime, movie_id: int, cinema_hall_id: int
```

Comment:
Place all arguments with type annotations on a new line to improve the readability.